src/pages/AlgoliaPOC.vue

```diff
@@ -274,6 +292,62 @@ export default {
 			// console.log(`New Search State: ${JSON.stringify(newDisjunctiveFacets)}`);
 			return newDisjunctiveFacets;
 		},
+		getRefinementCounts() {
+			console.log(this.$refs.sectorRefinements);
+			console.log(this.$refs.sectorRefinements.$el);
+			// well it takes a moment for these to show up...
+			// > Whats the right way to wait and do this? To monitor it based on the component instance
+			// > May be able to watch these or some other property to detect state changes
+			setTimeout(() => {
+				console.log(this.$refs.sectorRefinements.items);
+			}, 1000);
```

Comment:
Initial block for gathering counts...it's gonna be tricky as these values take a moment to load before being available.